Add Pirate Clipper, Cutthroat (#8121)
endless-sky/endless-sky: data/human/ships.txt

ship "Cutthroat"
	sprite "ship/pirate cutthroat"
	thumbnail "thumbnail/pirate cutthroat"
	attributes
		category "Light Warship"
		"cost" 1100000
		"shields" 2800
		"hull" 600
		"required crew" 4
		"bunks" 6
		"mass" 171

> 		"drag" 3

Reviewer: ```suggestion
		"drag" 3.8
```
Considering the ship is *gaining* outfit, engine, and mass (even visibly), I don't think the drag should be *lower*, but a tiny bit higher. This shouldn't be a problem considering its new engine and afterburner outfit.

Author: Eh, depending on what drag actually is. Is it just built-in speed limiter for the ship or is it a law of the universe. Also Marauders have zero drag change despite all the ships being heavier *also* a big clunky and "slow" Marauder Engine Leviathan literally outruns interceptor and light warships somehow, it is utterly ridiculous that its top speed jumps from 350 to 1000. The Cutthroat is supposed to be one of the fastest ship around and I think it definitely shouldn't be caught up so easily by such a big ship. (Even right now default variant is still slower)

Reviewer: Fair enough.

Reviewer 2: I think there's a strong case to be made that all (or at least most) of the marauder ships need to have their drag boosted a bit.

		"heat dissipation" .8
		"fuel capacity" 500

> 		"cargo space" 37

Reviewer: ```suggestion
		"cargo space" 27
```
Need to take this down even more considering it got 31 more engine and 22 more outfit.

		"outfit space" 282
		"weapon capacity" 60
		"engine capacity" 111
		weapon
			"blast radius" 36
			"shield damage" 360
			"hull damage" 180
			"hit force" 540
	outfits
		"Twin Modified Blaster" 2
		"Torpedo Pod" 2
		"Torpedo Storage Rack"
		"Torpedo" 21
		
		"Fission Reactor"
		"LP036a Battery Pack"
		"Water Coolant System"
		"D23-QP Shield Generator"
		
		"A370 Atomic Thruster"
		"A375 Atomic Steering"
		"Volcano Afterburner"
		"Hyperdrive"
		
	engine -22 47

> 	engine 22 47

Reviewer: Where extra engine flare.

	gun -21.5 -18 "Torpedo Pod"
	gun 21.5 -18 "Torpedo Pod"
	gun -11 -8.5 "Twin Modified Blaster"
	gun 11 -8.5 "Twin Modified Blaster"
	leak "leak" 60 50
	leak "flame" 50 80
	explode "tiny explosion" 15
	explode "small explosion" 20
	explode "medium explosion" 10
	"final explode" "final explosion small"

> 	description "Cutthroat is what the pirates call this Clipper with extra engine pods bolted on to it, with the sole purpose of maximizing speed to outrun all but the most fine-tuned ship in the sky."

Reviewer 3: ```suggestion
	description `Pirates call this clipper the "Cutthroat." It has extra engine pods bolted on to it, with the sole purpose of maximizing speed to outrun all but the most fine-tuned ship in the sky.`
```

Reviewer 3: Note: The reason I am changing the quotations to something else is because it wouldn't work since I put cutthroat in quotes.

ship "Cutthroat"
	sprite "ship/pirate cutthroat"
	thumbnail "thumbnail/pirate cutthroat"
	attributes
		category "Light Warship"
		"cost" 1100000
		"shields" 2800
		"hull" 600
		"required crew" 4
		"bunks" 6

> 		"mass" 171

Reviewer 2: ```suggestion
		"mass" 185
```
An extra 21 mass does not seem nearly enough to account for a 38% increase in engine space, especially since this ship is 21 lighter thanks to the cargo-to-outfit space conversion. This 21 merely accounts for the change in the outfitting space, and should have significantly more to account for the two big engine nacelles bolted onto the wings.

Author: Note they also did carve out bunks to upgrade so it's not just cargo space to outfit space but include the bunk area as well. The mass I added was accounting mostly for the added on engine pods.

> 		"drag" 3

Reviewer 2: ```suggestion
		"drag" 3.5
```

The fact that this has 38% more engine space than the regular clipper should be more than enough of a speed boost without also reducing its drag by 22%. A tweak? Sure, seems appropriate. A 22% drop isn't a tweak, that's completely replacing the core structural members with something stronger. Reasonable on a low performing ship, but the Clipper's already pretty much top of its class as a high-performance ship. I doubt it has anything less than humanity's best already.

Author: Yet Marauders somehow find ways to improve even already top of the line ships like Falcons with little change to external appearance. Sure Southbound shipyard is a capable designer and manufacturer but without doubt there will be cuts so it's more economical to build just like every product today so there will be ways that things can be improved.

Reviewer 2: I'm not saying that this can't have an improvement, I'm just positing that I don't think a 22% drop in drag is reasonable. The 3.5 I suggested *is* a improvement over the clipper's 3.7. So even if a pilot limited themselves to the same stock equipment as on a regular clipper, they'll still outpace it fairly easily. Throw in the fact that this has 138% of a clipper's engine capacity, and, well, there's zero question that this will be massively faster. This will be massively faster than a regular clipper even if the drag was left entirely alone.

Author: I'm afraid it wouldn't be enough to actually outrun most MELeviathan but turns out it still doesn't without reducing further to 2.6 or something. I'll bump it up to 3.4 and go nerf MELeviathan somehow.

Reviewer 2: If the marauder Leviathan can outrun this Cuthroat, then the Leviathan needs to be nerfed. Likewise for any other marauder heavy warship. Top speed isn't everything, though. What's the acceleration look like? This thing has ~370 acceleration, which vastly outperforms every interceptor and even fighter, with the finch coming in at a lowly ~290 in comparison. (just looking at the in-game shipyards. I don't have the marauder ships edited into this game currently)

Author: Acceleration is proportional to thrust and mass, there's little mass increase with Leviathan so MELevi still have more acceleration than everything else.

		"heat dissipation" .8
		"fuel capacity" 500
		"cargo space" 27

> 		"outfit space" 282

Reviewer 2: So, it loses 43 cargo to gain 22 outfit space? That's actually fairly reasonable.

		"weapon capacity" 60
		"engine capacity" 111
		weapon
			"blast radius" 36
			"shield damage" 360
			"hull damage" 180
			"hit force" 540
	outfits
		"Twin Modified Blaster" 2
		"Torpedo Pod" 2
		"Torpedo Storage Rack"
		"Torpedo" 21
		
		"Fission Reactor"
		"LP036a Battery Pack"
		"Water Coolant System"
		"D23-QP Shield Generator"
		
		"A370 Atomic Thruster"
		"A375 Atomic Steering"
		"Volcano Afterburner"
		"Hyperdrive"
		
	engine -22 47
	engine 22 47
	engine -13 43.5 .7
	engine 13 43.5 .7
	gun -21.5 -18 "Torpedo Pod"
	gun 21.5 -18 "Torpedo Pod"
	gun -11 -8.5 "Twin Modified Blaster"
	gun 11 -8.5 "Twin Modified Blaster"
	leak "leak" 60 50
	leak "flame" 50 80
	explode "tiny explosion" 15
	explode "small explosion" 20
	explode "medium explosion" 10
	"final explode" "final explosion small"

> 	description `Pirates call this heavily modified clipper the "Cutthroat." It has extra engine pods bolted on to it, with the sole purpose of maximizing speed to outrun all but the most fine-tuned ship in the sky.`

Reviewer 4: ```suggestion
	description `Pirates call this heavily modified clipper the "Cutthroat." It has extra engine pods bolted onto it, with the sole purpose of maximizing speed to outrun all but the most fine-tuned ships in the sky.`
```

ship "Cutthroat"
	sprite "ship/pirate cutthroat"
	thumbnail "thumbnail/pirate cutthroat"
	attributes
		category "Light Warship"
		"cost" 1100000
		"shields" 2800
		"hull" 600
		"required crew" 4
		"bunks" 6
		"mass" 175

> 		"drag" 3.1

Reviewer 2: It took me less than 5 seconds to get the standard Cuthroat to >1200 max speed, while still having 97 turning. If I'm willing to drop turn down to 52, I can get the max speed up to almost 1585 with 634 acceleration!!! Seems a tad extreme for a human freighter turned warship when a dedicated racing ship can't even approach that.

		"heat dissipation" .8
		"fuel capacity" 500
		"cargo space" 27
		"outfit space" 282
		"weapon capacity" 60
		"engine capacity" 111
		weapon
			"blast radius" 36
			"shield damage" 360
			"hull damage" 180
			"hit force" 540
	outfits
		"Twin Modified Blaster" 2
		"Torpedo Pod" 2
		"Torpedo Storage Rack"
		"Torpedo" 21
		
		"Fission Reactor"
		"LP036a Battery Pack"
		"Water Coolant System"
		"D23-QP Shield Generator"
		"Laser Rifle" 5
		
		"A370 Atomic Thruster"
		"A375 Atomic Steering"
		"Volcano Afterburner"
		"Hyperdrive"
		
	engine -22 47
	engine 22 47
	engine -13 43.5 .7
	engine 13 43.5 .7
	gun -21.5 -18 "Torpedo Pod"
	gun 21.5 -18 "Torpedo Pod"
	gun -11 -8.5 "Twin Modified Blaster"
	gun 11 -8.5 "Twin Modified Blaster"
	leak "leak" 60 50
	leak "flame" 50 80
	explode "tiny explosion" 15
	explode "small explosion" 20
	explode "medium explosion" 10
	"final explode" "final explosion small"

> 	description `Pirates call this heavily modified clipper the "Cutthroat." It has extra engine pods bolted onto it, with the sole purpose of maximizing speed to outrun all but the most fine-tuned ships in the sky.`

Reviewer 5: ```suggestion
	description `Pirates call this heavily modified Clipper the "Cutthroat." It has extra engine pods bolted onto it, with the sole purpose of maximizing speed to outrun all but the most fine-tuned ships in the sky.`
```
I think, technically it is also correct as is, but this change is needed if we are referring directly to the base ship the way I think we should here ;)

> 	description `Pirates call this heavily modified clipper the "Cutthroat." It has extra engine pods bolted onto it, with the sole purpose of maximizing speed to outrun all but the most fine-tuned ships in the sky.`

Reviewer 5: ```suggestion
	description `Pirates call this heavily modified Clipper the "Cutthroat." With an extra pair of engine pods bolted on for the sole purpose of maximizing speed, it can catch or outrun all but the most fine-tuned ships in the sky.`
```
Purely stylistic switch ups :)
